playbook/app/pb_kits/playbook/pb_table/docs/_table_sm.md

```diff
@@ -1 +1,2 @@
 Use table size `"sm"` when data density is a priority. Smaller row height enables the user to view more data without the need for scrolling.
+Table can leverage the max-width property. Learn more in our <a href="https://playbook.powerapp.cloud/visual_guidelines" target="_blank">visual guidelines.</a>
```

Comment:
Add a return here, between the two sentences. Just the code, don't worry about updating the screenshot.

Reply:
So it would look like this?
Use table size `"sm"` when data density is a priority. Smaller row height enables the user to view more data without the need for scrolling.
return 
Table can leverage the max-width property. Learn more in our <a href="https://playbook.powerapp.cloud/visual_guidelines" target="_blank">visual guidelines.</a>

Reply:
Or did you mean a break between the sentences ?

Reply:
So it would look more like this?
<img width="890" alt="Screen Shot 2021-01-08 at 12 12 49 PM" src="https://user-images.githubusercontent.com/64423298/104044118-d7b22a80-51aa-11eb-94e6-66f914261eff.png">